### PR TITLE
Add ABOUT.md, LEARNING.md, and RESOURCES.md

### PR DESCRIPTION
**Pull request by outkaj**
Adding information that was missing on [the Haskell section of the website] (http://exercism.io/languages/haskell).

### CODE REVIEW

File: docs/RESOURCES.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+##Recommended Learning Resources
```
Comment (kytrinyx) [inline docs/RESOURCES.md]: The resources page already has a header, so this would make a duplicate.

File: docs/LEARNING.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Please see the "Useful Haskell Resources" tab.
```
Comment (kytrinyx) [inline docs/LEARNING.md]: Learning should be about learning the language from scratch, whereas resources is more about tools, references, and resources that you might use while developing in Haskell.

**Comment by kytrinyx** [comment]
I'd like to have something about what kinds of problems Haskell is used to solve in the `ABOUT.md`.

**Comment by rbasso** [comment]
This PR is inactive for 6 months and most of the proposed content was, meanwhile, included in the repository by #93 and #97.

Considering that...

- any old changes would have to be completely rewritten to merge with the current *docs*
- the content proposed is similar to what we currently have
- half an year has elapsed without answer from the author.

... I'll close this PR.

**Pull request closed without merging**